### DISCUSSION TITLE
Getting Deep Link Source Application

### DISCUSSION THREAD

**Discussion by Brandi-Youreka** [ionic-team/capacitor#5836] [Q&A]
TLDR: Is there a way to get the url from the application that is deep linking into my React application?

In my React application I would like to allow the user to

-  Deep link into my application
- Complete some tasks
- Deep link out of my application back to the source application that they were initially in

Branch.io does this by exposing a `retURL` key in their data object. This key hold the url of the application that deep linked into my application. Is there a way to get the url from the application that is deep linking into my React application or code I can modify to get this information?
